CHANGES.md

```diff
@@ -1,3 +1,44 @@
+## edge-19.8.2
+
+**Significant Update**
+
+This edge release introduces the new Linkerd control plane Helm chart, named
+linkerd2. Helm users can now install and remove the Linkerd control plane by
+using the `helm install` and `helm delete` commands. Proxy injection also now
+uses Helm charts.
+
+No changes were made to the existing `linkerd install` behavior.
+
+For detailed installation steps using Helm, see the notes for [PR #3146]
+(https://github.com/linkerd/linkerd2/pull/3146).
+
+* CLI
+  * Moved `linkerd top` and `linkerd profile --tap` to the new tap APIService
+  * Introduced parent resources for all tappable subresources in the tap
+    APIService, enabling commands such as `kubectl auth can-i`
+* Controller
+  * Introduced a new ClusterRole, `linkerd-linkerd-tap-admin`, which gives
+    cluster-wide tap privileges. Also introduced a new ClusterRoleBinding,
+    `linkerd-linkerd-web-admin`, which binds the `linkerd-web` service account
+    to the new tap ClusterRole. This ClusterRoleBinding is enabled by default,
+    but may be disabled via a new `linkerd install` flag,
+    `--restrict-dashboard-privileges`
```

Comment:
I think we should make a mention of the effects of this option. Seems like certain things on the web UI won't work with `--restrict-dashboard-privileges=true`. I will defer to @siggy or @grampelberg for wording.

Reply:
Agree about focusing on the end-user effect. Consider moving mention of `--restrict-dashboard-privileges` to the CLI section, something like:
```
Introduced `--restrict-dashboard-privileges` flag to `linkerd install` command, to restrict the Linkerd Dashboard's default privileges to disallow Tap
```

Reply:
Thanks for the suggestions @ihcsim @siggy. To clarify, should we keep the background information about the new ClusterRole in the `Controller section`, or delete that portion entirely and only mention this in the `CLI` section?

Reply:
Ah good point. Yeah I think it's fine to mention the new ClusterRole in the `Controller section`. It's relevant because if users want to give themselves (or their serviceaccounts) tap privileges, they can use that new `linkerd-linkerd-tap-admin` role.